content/components/action-list.mdx

```diff
@@ -0,0 +1,221 @@
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/121966246-226c3b80-cd23-11eb-856b-32bde42cb60f.png" />
+                <Caption>Use leading visuals to represent system sections, features, or options.</Caption>
+            </Box>
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/121966249-2304d200-cd23-11eb-9bbf-fbc1ef2773b1.png" />
+                <Caption>Use leading visuals in important menu items.</Caption>
+            </Box>
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/121966449-8c84e080-cd23-11eb-930f-9397ae2ea433.png" />
+                <Caption>Use leading visuals to represent content types.</Caption>
+            </Box>
+        </Flex>
+    </div>
+    <div>
+        <Heading fontSize={3}>Leading visual</Heading>
+        <Flex as="p">Leading visuals are optional and appear at the start of an item. They can be octicons, avatars, and other custom visuals that fit a small area.</Flex>
+        <Flex as="p">When listing system sections, features, or options, use leading visuals to improve the items' scannability. In user-generated objects, they can help indicate the item's content type and status.</Flex>
+        <Flex as="p">Depending on the context, displaying a leading visual may not be necessary. For example, a list of branches in a select panel may not need repeated icons if the surrounding UI provides enough hints about its content type.</Flex>
+    </div>
+    <div>
+        <Flex flexDirection="column" justifyContent="center">
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/124040519-eb03bd00-d9b9-11eb-94ec-2653cb2e2278.png" />
+                <Caption>A right arrow as a trailing visual indicates there are more options to choose after selecting an item.</Caption>
+            </Box>
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/124064900-9298e380-d9ea-11eb-8aa9-dae40ece9d5b.png" />
+                <Caption>Trailing text with custom styling to indicate diff change.</Caption>
+            </Box>
+        </Flex>
+    </div>
+    <div>
+        <Heading fontSize={3}>Trailing visual and trailing text</Heading>
+        <Box as="p">Trailing visual and trailing text can display auxiliary information. They're placed at the right of the item, and can denote status, keyboard shortcuts, or be used to set expectations about what the action does.</Box>
+        <Box as="p">Note these side visuals don't have dedicated interaction targets.</Box>
+        <Box as="p">Use an `arrow-right` octicon in menus to indicate the action will open more options, such as in a nested context. Use an `edit` octicon to indicate the item is going to be edited after clicking it.</Box>
```

Comment:
We need figure out why these code blocks aren't rendering in the preview and either fix or use normal text.

Reply:
In MDX, text is not parsed as markdown unless it's surrounded by newlines. So backticks won't be treated as inline code unless the containing text block is surrounded by newlines. Give this a try:

```suggestion
        <Box as="p">
       
Use an `arrow-right` octicon in menus to indicate the action will open more options, such as in a nested context. Use an `edit` octicon to indicate the item is going to be edited after clicking it.

        </Box>
```

Reply:
@colebemis It works, thanks! 🥳